parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -132,7 +132,17 @@ write_files:
   encoding: gzip
   owner: root
   content: !!binary |
-    {{GetVariableProperty "cloudInitData" "migPartitionScript"}}  
+    {{GetVariableProperty "cloudInitData" "migPartitionScript"}}
+
+- path: /etc/systemd/system/nvidia-device-plugin.service.d/10-mig_strategy.conf
+  permissions: "0644"
+  owner: root
+  content: |
+    [Service]
+    Environment="MIG_STRATEGY=--mig-strategy single"
+    ExecStart=
```

Comment:
Is this empty ExecStart needed?

Reply:
yes, you can't override execstart directly otherwise as I just learned :)